src/framework/audio/internal/fx/fxresolver.cpp

```diff
@@ -46,7 +46,7 @@ std::vector<IFxProcessorPtr> FxResolver::resolveMasterFxList(const AudioFxChain&
         AudioFxChain fxChainByType;
 
         for (const auto& fx : fxChain) {
-            if (resolver.first == fx.second.type()) {
+            if (resolver.first == fx.second.type() || fx.second.type() == AudioFxType::Undefined) {
```

Comment:
it would be great to add a comment about `fx.second.type() == AudioFxType::Undefined`

Reply:
Agreed, will add in next PR with audio signal changes optimization